templates/cloud/index.html

```diff
@@ -1,250 +1,364 @@
+                In addition to the compute costs, there can be unexpected ancillary costs like egress (taking your data out of the cloud)
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row--50-50">
+      <hr class="is-muted" />
+      <div class="col"></div>
+      <div class="col">
+        <p>
+          <a href="/download/cloud">Learn more about Ubuntu on Public Clouds&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+    </div>
+  </section>
 
-<section class="p-strip--light">
-  <div class="row u-equal-height">
-    <div class="col-8">
-      <h2>What is a hybrid cloud?</h2>
-      <p>A <a href="/cloud/hybrid-cloud">hybrid cloud</a> architecture is one that combines the usage of a private cloud and one or more public cloud services with a workload orchestration engine between the platforms.</p>
-    </div>
-    <div class="col-4 u-hide--small u-vertically-center u-align--center">
-      {{
-        image(
-         url="https://assets.ubuntu.com/v1/45120edb-Hybrid+Cloud.svg",
-         alt="Hybrid cloud",
-         width="190",
-         height="130",
-         hi_def=True,
-         loading="auto",
-        ) | safe
-      }}  
-    </div>
-  </div>
-  <div class="row u-equal-height">
-    <div class="col-5">
-      <p><b>Advantages:</b></p>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">A hybrid strategy provides maximum OpEx efficiency when executed in the right way.</li>
-        <li class="p-list__item is-ticked">Hybrid clouds are ideal for workloads with diverse requirements.</li>
-        <li class="p-list__item is-ticked">Prevents vendor-lock in as organisations are not tied to one cloud vendor.</li>
-      </ul>
-    </div>
-    <div class="col-5">
-      <p><b>Disadvantages:</b></p>
-      <ul class="p-list">
-        <li class="p-list__item is-crossed">Workload orchestration across various cloud platforms can be challenging.</li>
-      </ul>
-    </div>
-  </div>
-  <div class="u-fixed-width">
-    <p>
-      <a href="/engage/hybrid-cloud-strategy-playbook">Read our hybrid cloud modernisation playbook&nbsp;&rsaquo;</a>
-    </p>
-  </div>
-</section>
+  <section class="p-section">
+    <div class="p-section--shallow">
+      <div class="row--50-50">
+        <hr class="p-rule" />
+        <div class="col">
+          <h2>What is a hybrid cloud?</h2>
+        </div>
+        <div class="col">
+          <p>
+            A <a href="/cloud/hybrid-cloud">hybrid cloud</a> architecture is one that combines the usage of a private cloud and one or more public cloud services with a workload orchestration engine between the platforms.
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-9 col-start-large-4 col-medium-5 col-start-medium-2">
+        <hr class="is-muted" />
+        <div class="row">
+          <div class="col-3 col-medium-2">
+            <h5>Advantages:</h5>
```

Comment:
```suggestion
            <h3 class="p-heading--5">Advantages:</h3>
```
Try to avoid skipping heading levels for better accessibility. So if the heading above is a `h2`, the next should be a 'h3'. We can then use CSS to get the appropriate styling. There's more information here: https://www.w3.org/WAI/tutorials/page-structure/headings/